index.html

```diff
@@ -11,15 +11,55 @@
     integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
     crossorigin="anonymous"
     referrerpolicy="no-referrer" />
+  <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@400;600&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
-<body>
-  <header>
-  </header>
-  <main>
-  </main>
+<body class="body-bgc">
+  <div class="row header-bgc">
+    <header class="column-full">
+      <div class="row container">
```

Comment:
You shouldn't have `container`s inside of rows or columns. You should make the `container` the most external layer here, right inside the element you apply the purple background color to. That way your purple background fills the whole width of the viewport, but the content is confined in the `container`